tests/kernels/attention/test_flashinfer_trtllm_decode_attention.py

```diff
@@ -0,0 +1,137 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+from typing import Optional
+
+import flashinfer
+import pytest
+import torch
+
+from vllm.platforms import current_platform
+
+FLOAT32_BYTES = torch.finfo(torch.float).bits // 8
+
+# KV Cache Layout for TRT-LLM
+# kv_cache_shape = (num_blocks, 2, num_kv_heads, page_size, head_dim)
+
+# Only supports - num_query_heads / num_kv_heads == 8 atm
```

Comment:
Do we need to check this in the kernel? I don't see a check

Reply:
Added a kernel check